aether-common-module/aether/common/conf/settings.py

```diff
@@ -232,6 +232,26 @@
 if os.environ.get('DJANGO_HTTP_X_FORWARDED_PROTO', False):    # pragma: no cover
     SECURE_PROXY_SSL_HEADER = ('HTTP_X_FORWARDED_PROTO', 'https')
 
+# Storage Configuration
+# ------------------------------------------------------------------------------
+
+CUSTOM_STORAGE = os.environ.get('DJANGO_REMOTE_STORAGE', '')
+
+if CUSTOM_STORAGE:   # pragma: no cover
+    INSTALLED_APPS += ['storages', ]
+    if CUSTOM_STORAGE.lower() == 's3':
+        DEFAULT_FILE_STORAGE = 'storages.backends.s3boto3.S3Boto3Storage'
+        AWS_AUTO_CREATE_BUCKET = True
+        AWS_ACCESS_KEY_ID = os.environ.get('AWS_ACCESS_KEY_ID', '')
```

Comment:
you should be able to simply remove `AWS_SECRET_ACCESS_KEY` and `AWS_SECRET_ACCESS_KEY`

Reply:
do you need to set `AWS_DEFAULT_REGION`?

Reply:
`AWS_DEFAULT_REGION` is not required